TestApp/package.json

```diff
@@ -18,6 +18,7 @@
     "react-native-dialog-component": "^0.2.17",
     "react-native-fs": "^2.9.12",
     "react-native-image-picker": "^0.26.7",
+    "react-native-modal-selector": "0.0.28",
```

Comment:
Is there something wrong with the built-in [Picker](https://facebook.github.io/react-native/docs/picker)?

Reply:
Yes, on ios it can't be collapsed. See [here.](https://stackoverflow.com/questions/41181683/react-native-ios-picker-is-always-open) It's not the bug, but the default's picker behavior.